README.client.rst

```diff
@@ -1,9 +1,35 @@
-Rucio Client Lite Package
+Rucio: Client Lite Package
 =========================
 
+Rucio is a project that provides services and associated libraries for allowing scientific
```

Comment:
I like having this sentence in every README, we should just wait for the confirmed one on Tuesday.